### PR TITLE
Add unit tests to ConfigApi

### LINKED ISSUE #63: Add unit tests to ConfigApi
Add unit testing to `ConfigApi` service. Something similar to the [TransactionApi ones](https://github.com/5afe/safe-client-gateway-nest/blob/main/src/datasources/transaction-api/transaction-api.service.spec.ts) should work.

### PR DESCRIPTION
**Pull request by Uxio0**
- Closes #63

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "typescript.preferences.importModuleSpecifier": "relative"
+  "typescript.preferences.importModuleSpecifier": "relative",
+  "[typescript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
+  }
```
Comment (fmrsabino) [inline .vscode/settings.json:6]: I would keep this in a separate PR just for keeping this one with the unit test.

What does this change do exactly?

Comment (Uxio0) [reply .vscode/settings.json]: I'm afraid it was vscode, I didn't realise

Comment (Uxio0) [reply .vscode/settings.json]: I've just undo this change

**Review by hectorgomezv — APPROVED** [review]
🚀

**Review by fmrsabino — APPROVED** [review]
